libs/polkadot-api-provider/src/webb-provider/vanchor-withdraw.ts

```diff
@@ -362,9 +367,9 @@ export class PolkadotVAnchorWithdraw extends VAnchorWithdraw<WebbPolkadot> {
       this.cancelToken.throwIfCancel();
       txHash = await tx.call(account.address);
     }
-
+    // TODO construct the right note
```

Comment:
What is wrong with the note?

Reply:
This was from a previous change will remove the comment